docs/user/notifications.rst

```diff
@@ -0,0 +1,45 @@
+.. _notifications:
+
+Notifications
+=============
+
+.. contents:: Content of this document
+    :local:
+    :depth: 2
+
+Artemis supports customizable web and email notifications. Users can enable and disable different notification types.
```

Comment:
Types of notification sounds more natural. Just like in [line 33](https://github.com/ls1intum/Artemis/blob/936154fc747be74638e2644952fdcaf0e48f93a9/docs/user/notifications.rst?plain=1?#L33)